src/main/io/vtx_smartaudio.c

```diff
@@ -1,906 +0,0 @@
-/*
- * This file is part of Cleanflight and Betaflight.
```

Comment:
This file should definitely not be removed.

Reply:
Sorrry bout that.